submissions/dafen173/pop-up2/index.html

```diff
@@ -0,0 +1,154 @@
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/calendar.png" alt="Calendar" />
+								<div class="captions">Calendar</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/cloud.png" alt="Cloud" />
+								<div class="captions">Cloud</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/store.png" alt="Store" />
+								<div class="captions">Store</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img
+									src="icons/translate.png"
+									alt="Translate"
+								/>
+								<div class="captions">Translate</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/music.png" alt="Music" />
+								<div class="captions">Music</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/movie.png" alt="Movie" />
+								<div class="captions">Movie</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/alarm.png" alt="Alarm" />
+								<div class="captions">Alarm</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/book.png" alt="Books"/>
+								<div class="captions">Books</div>
+							</a>
+						</li>
+
+					</ul>
+
+
+					<input type="checkbox" id="buttonMore" class="MoreApps" tabindex="7">
+					<label for="buttonMore" class="MorePopUp">More</label>
+
+					<hr class="line">
+
+					<ul class="hiddenBlock">
+						
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/notes.png" alt="Notes" />
+								<div class="captions">Notes</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/photo.png" alt="Photo" />
+								<div class="captions">Photo</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/timer.png" alt="Timer" />
+								<div class="captions">Timer</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/mail.png" alt="Mail" />
+								<div class="captions">Mail</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem">
+							<a href="#">
+								<img src="icons/chat.png" alt="Chat" />
+								<div class="captions">Chat</div>
+							</a>
+						</li>
+
+						<li class="popupIconsItem"></li>
+					</ul>
+				</div>
+
+
+				<div class="menuItem" tabindex="4">
+					<img src="icons/bell.png" alt="Notifications" />
+				</div>
+				<div class="menuItem" tabindex="5">
+					<img class="ava" src="icons/avatar.png" alt="Avatar" />
+				</div>
+
+
+				<div class="triangle">
+					<div class="empty"></div>
+				</div>
+
+
+			</div>
+		</nav>
+	</body>
+</html>
```

Comment:
If you see this symbol 
![empty line](https://user-images.githubusercontent.com/10871447/107821483-3871eb80-6d85-11eb-9102-b67761d7511c.png)   that means the empty line needs at the end of the file. This line is necessary for webpack (gulp, etc). I will add this symbols in these case and merge